### PR TITLE
Hotfix/1.4.2

### PR DESCRIPTION
**Pull request by turegjorup**
Update drupal core and contrib
* Update `email_registration` from `1.0.0-rc5` to `1.1`
* Replace abandoned `flood_unblock` with `flood_control`
* Update `purge_users` from `1.9` to `2.0`
* Update `slick_entityreference` from `dev-1.x 25b6a55` to `1.1`
* Update `swiftmailer` from `1.0.0-beta2` to `2.0` (Note that the native `php` option has been removed: https://www.drupal.org/project/swiftmailer/issues/3143292)
* Update `user_registrationpassword` from `alpha3` to `alpha5`

### CODE REVIEW

File: config/sync/core.entity_view_mode.node.full.yml [hunk 1]
```diff
@@ -7,6 +7,6 @@ dependencies:
 _core:
   default_config_hash: ElrtInxGjZd7GaapJ5O9n-ugi2hG2IxFivtgn0tHOsk
 id: node.full
-label: 'Full content'
+label: 'Fuldt indhold'
```
Comment (cableman) [inline config/sync/core.entity_view_mode.node.full.yml:10]: Changes in language?

File: config/sync/litteratursiden.settings.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ region_wells:
   footer: ''
 table_responsive: '-1'
 table: ''
-background_path: 'public://stine_pilgaard_topbilledei_0.jpg'
+background_path: 'public://001 Foto- Tale Hendnes 2020-25 (1)_0.jpg'
```
Comment (cableman) [inline config/sync/litteratursiden.settings.yml:24]: Do we override the default background image on release if changed by editors?

**Review by cableman — CHANGES_REQUESTED** [review]

File: config/sync/litteratursiden.settings.yml (hunk 2 above)
Comment (turegjorup) [reply config/sync/litteratursiden.settings.yml:24]: Have pulled the `prod` db before update, so this is the current setting. But could be argued that we should use config-split or similar to exclude the `background_path`

File: config/sync/core.entity_view_mode.node.full.yml (hunk 1 above)
Comment (turegjorup) [reply config/sync/core.entity_view_mode.node.full.yml:10]: Nope. No clue why

**Comment by turegjorup** [comment]
Branch deployed to `dev` without errors. Email sign-up tested and works.

**Pull request closed without merging**